Text/AbbyyOCR/AbbyyOCR.cs

```diff
@@ -0,0 +1,257 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using AzureCognitiveSearch.PowerSkills.Common;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using Abbyy.CloudSdk.V2.Client.Models;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net.Http;
+using Abbyy.CloudSdk.V2.Client;
+using Abbyy.CloudSdk.V2.Client.Models.RequestParams;
+using Abbyy.CloudSdk.V2.Client.Models.Enums;
+using System.Net;
+using Polly.Extensions.Http;
+using Polly;
+using System.Collections.Concurrent;
+
+namespace AbbyyOCR
+{
+    public static class AbbyyOCR
+    {
+        private const string ApplicationId = @"[Enter ABBYY Application ID]";
+        private const string Password = @"[Enter ABBYY OCR Password]";
+        private const string ServiceUrl = "[Enter ABBYY Service URL such as https://cloud-westus.ocrsdk.com]";
+
+        private static int _retryCount = 3;
+        private static int _delayBetweenRetriesInSeconds = 3;
```

Comment:
Do we want to make that configurable, and use those values as defaults?

Reply:
I'm going to decline this one as it follows the ABBYY sample recommendataions

Reply:
I am following the ABBYY documentation on this one, so I am leaning towards keeping it.  Let me know if you feel strongly that this should be changed.

Reply:
Yes, as mentioned in email, settings instead of constants is the only way to allow deployment from GitHub (in addition to being cleaner).